src/lib/holocene/modal.svelte

```diff
@@ -137,15 +137,11 @@
       {/if}
     </div>
     <div class="flex items-center justify-end space-x-2 p-6">
-      <Button
-        thin
-        variant="secondary"
-        disabled={loading}
-        on:click={handleCancel}>{cancelText}</Button
+      <Button variant="secondary" disabled={loading} on:click={handleCancel}
```

Comment:
Do we want a `size="xs"` here to replace `thin`?

Reply:
Tbh I think the buttons look better here unmodified 🤷

|default|xs|
|---|---|
|<img width="546" alt="Screenshot 2023-08-30 at 12 57 38 PM" src="https://github.com/temporalio/ui/assets/11775628/ad1a99a7-995b-40c4-a97d-798914deb38f">|<img width="546" alt="Screenshot 2023-08-30 at 12 57 21 PM" src="https://github.com/temporalio/ui/assets/11775628/60f52207-97e8-4482-ab26-4f779de39b08">|